API/MoodisAPI/apiMoodis/apiMoodis/Models/ImageInfo.cs

```diff
@@ -0,0 +1,32 @@
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated from a template.
+//
+//     Manual changes to this file may cause unexpected behavior in your application.
+//     Manual changes to this file will be overwritten if the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace apiMoodis.Models
+{
+    using System;
+    using System.Collections.Generic;
+    
+    public partial class ImageInfo
+    {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
+        public ImageInfo()
+        {
+            this.Emotions = new HashSet<Emotion>();
+        }
+    
+        public string Id { get; set; }
+        public string UserId { get; set; }
+        public string ImagePath { get; set; }
+        public string DateAsString { get; set; }
+    
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
+        public virtual ICollection<Emotion> Emotions { get; set; }
+        public virtual User User { get; set; }
```

Comment:
is `user` necessary if we have its id?

Reply:
This is pre generated based on the structure of the database. Since we have a "one to many" relationship this is how it references it. One end holds the referenced object while the other has a list. Default initialization is lazy so these values remain null unless you specify otherwise so there would be no problems that concern performance. This would be the same case as with your previous comment about ImageInfo object

Reply:
Back End master Aurimas 🥇